api/src/models/FedoraObject.ts

```diff
@@ -158,11 +158,12 @@ export class FedoraObject {
             return acc + agentXml;
         }, "");
         const documentXml = `
+            <?xml version="1.0" encoding="UTF-8"?>
             <METS:metsHdr xmlns:METS="http://www.loc.gov/METS/" CREATEDATE="${xmlescape(
                 createDate || new Date().toISOString()
-            )}"${
-            createDate ? ` LASTMODDATE="${new Date().toISOString()}"` : ""
-        } RECORDSTATUS="">${agentsXml}</METS:metsHdr>
+            )}"${createDate ? ` LASTMODDATE="${new Date().toISOString()}"` : ""}${
+            recordStatus ? ` RECORDSTATUS="${recordStatus}"` : ""
```

Comment:
We should probably xmlescape the record status to be on the safe side (unless it's being passed through in pre-escaped form). I think it's unlikely this will ever contain special characters, but better safe than sorry.

Reply:
mde change.